src/bindings/nodejs/bin/valhalla-cli.js

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env node
+
+const { spawn } = require('child_process');
+const path = require('path');
+const fs = require('fs').promises;
+const fsSync = require('fs');
+const { getBinaryDir } = require('../lib/binary-path');
+
+const binaryDir = getBinaryDir(path.join(__dirname, '..'));
+const packageRootDir = path.join(__dirname, '..');
+
+ 
+
+async function fileExists(filePath) {
+    try {
+        await fs.access(filePath);
+        return true;
+    } catch {
+        return false;
+    }
+}
+
+// when we publish the package, we lose the executable permissions, so we need to ensure they are set
```

Comment:
huh that's strange.. you sure it's not because of some quirk in the manual packaging?